src/main/kotlin/org/danilopianini/gradle/gitsemver/GitSemVer.kt

```diff
@@ -17,12 +16,21 @@ class GitSemVer : Plugin<Project> {
             val extension = project.createExtension<GitSemVerExtension>(GitSemVerExtension.EXTENSION_NAME, project)
             project.afterEvaluate {
                 with(extension) {
-                    assignGitSemanticVersion()
+                    properties["forceVersion"]?.let {
```

Comment:
Instead of hardwiring a string, it would be better to have a `Property` in the extension with a default name

Reply:
I tried to address your comment.
Let me know if it is what you meant!